### DISCUSSION TITLE
smtp gmail not working : certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by bobwatcherx** [pocketbase/pocketbase#243] [Q&A]
i am using google smtp and trying to create a new user with verification email

but in my inbox there is no email
and i checked the log
with an error message like this:

**certificate signed by unknown authority**

I tried the smtp tester on the web https://dnschecker.org/smtp-test-tool.php and it worked
but not working for pocketbase

**Comment by ganigeorgiev** [comment]
Have you enabled "less secure apps" for your gmail account?
Are you running the application from a docker container?
How do you access the application - from localhost or a public address?
What "TLS Encryption" setting are you using - StartTLS or "Always"?

I'm not sure what dnschecker test and how it works, but **I don't recommend entering your auth credentials in third-party websites**.

If you need a SMTP server just for test purposes, I often use - https://mailtrap.io/.

**Comment by carlosdlp** [comment]
Hello!,

Im currently having the same issue, when I run pocketbase on my local machine theres no error, but when a set up pocketbase on a server using a docker container and accessing via public address I get the error "Failed to send the test email. Raw error: 
x509: certificate signed by unknown authority."

**Reply by ganigeorgiev** [reply]
@carlosdlp If you are running PocketBase in a docker with stripped base image then most likely you have to install the `ca-certificates` package. You could check also https://github.com/pocketbase/pocketbase/discussions/684 and https://github.com/pocketbase/pocketbase/discussions/801.

**Reply by carlosdlp** [reply]
Lol, thanks you for the quick response, its working now!!
